packages/remediations/src/steps/reviewSystems.js

```diff
@@ -0,0 +1,160 @@
+
+    const allNewSystems = dedupeArray(issues.reduce((acc, curr) => [
+        ...acc,
+        ...(curr.systems || [])
+    ], [ ...systems ]));
+
+    const allSystems = playbook ? dedupeArray(playbook.issues?.reduce((acc, curr) => [
+        ...acc,
+        ...(curr.systems?.map(system => system.id) || [])
+    ], [ ...allNewSystems ])) : allNewSystems;
+
+    const onRefresh = (options, callback) => {
+        if (!callback && inventory && inventory.current) {
+            inventory.current.onRefreshData(options);
+        } else if (callback) {
+            callback(options);
+        }
+    };
+
+    const onSelect = (selected) => {
+        input.onChange(selected);
+    };
+
+    const onSelectRows = (value) => {
+        dispatch({
+            type: TOGGLE_BULK_SELECT,
+            payload: value
+        });
+    };
+
+    const fetchSystemsInfo = useCallback(async (_i, config) => {
+        const systems = (allSystems || []).slice((config.page - 1) * config.per_page, config.page * config.per_page);
+        const data = await inventoryApi?.getEntities(systems, { ...config, hasItems: true, page: 1 }, true);
+        return {
+            ...data,
+            total: allSystems.length,
+            page: config.page,
+            // eslint-disable-next-line camelcase
+            per_page: config.per_page
+        };
+    }, [ allSystems ]);
+
+    return (
+        <Stack hasGutter>
+            <StackItem>
+                <TextContent>
+                    <Text>
+                        Review and optionally exclude systems from your selection.
+                    </Text>
+                </TextContent>
+            </StackItem>
+            <StackItem>
+                {
+                    <Router>
+                        <InventoryTable
+                            hideFilters={{ all: true }}
+                            noDetail
+                            variant="compact"
+                            showTags
+                            onRefresh={onRefresh}
+                            ref={inventory}
+                            getEntities={(_i, config) => fetchSystemsInfo(_i, config)}
+                            onLoad={({ mergeWithEntities, api }) => {
+                                registry.register(mergeWithEntities(entitiesReducer(onSelect, input.value)));
+                                setInventoryApi(() => api);
+                            }
+                            }
+                            bulkSelect={{
+                                id: 'select-systems',
+                                count: selected.length,
+                                items: [{
+                                    title: 'Select none (0)',
+                                    onClick: () => onSelectRows(false)
+                                },
+                                {
+                                    ...loaded && rows && rows.length > 0 ? {
```

Comment:
You don't have to check for `rows`
```suggestion
                                    ...loaded && rows.length > 0 ? {
```

Reply:
@karelhala thank you, updated :-)